src/RoleObjects/RadioData.js

```diff
@@ -110,4 +110,13 @@ export default class RadioData extends InputTagData {
       this._displayObject.dispatchEvent(event);
     }
   }
+
+  /**
+   * Event listener for change events
+   * @access protected
+   */
+  _onChange() {
+    const event = new createjs.Event('change');
+    this._displayObject.dispatchEvent(event);
```

Comment:
If this is event that code that uses CAM can receive, which looks to be the case given the test app update, then please update the radio role's events in Events table in readme.md.